mantle/platform/qemu.go

```diff
@@ -51,8 +51,6 @@ import (
 	"github.com/coreos/coreos-assembler/mantle/system"
 	"github.com/coreos/coreos-assembler/mantle/system/exec"
 	"github.com/pkg/errors"
-
-	"golang.org/x/sys/unix"
```

Comment:
It should be part of the first commit, squash the commits to have only one commit.